etc/restic/_global.env.template

```diff
@@ -17,4 +17,4 @@ export B2_ACCOUNT_ID="<b2-account-id>"   # TODO fill with your account info
 export B2_ACCOUNT_KEY="<b2-account-key>" # TODO fill with your account info
 
 # How many network connections to set up to B2. Default is 5.
-export B2_CONNECTIONS=10
+export B2_CONNECTIONS="10"
```

Comment:
Quoting variables makes the parsing part easier.

Reply:
Yes, but we can not rely on that users will adhere to a specific style, anything that is bash is allowed